test/libsolidity/smtCheckerTests/try_catch/try_nested_1.sol

```diff
@@ -0,0 +1,21 @@
+pragma experimental SMTChecker;
+contract C {
+	int public x;
+
+	function f() public view {
+		int y = 42;
+		bool success = false;
+		try this.x() returns (int v) {
+			y = v;
+			try this.x() returns (int w) {
+				success = true;
+				y = w;
+			}
+			catch {}
+		} catch {}
+		assert(!success || y == x); // should hold
```

Comment:
```suggestion
		assert(!success || y == w); // should hold
```
Did you mean this?

Reply:
No, that is not possible, `w` is local to the inner `try`'s success clause, it cannot be accessed at this point.

Reply:
Should I change this to  using `choice`, similar to the other tests?

Reply:
Ah sorry, just noticed that it calls `x` returned as `v`, so it's fine